src/dodal/devices/fluorescence_detector_motion.py

```diff
@@ -1,16 +1,14 @@
 from ophyd_async.core import StandardReadable, StrictEnum
-from ophyd_async.epics.core import epics_signal_r
+from ophyd_async.epics.core import epics_signal_rw
 
 
-class FluorescenceDetectorControlState(StrictEnum):
+class InOut(StrictEnum):
     OUT = "Out"
     IN = "In"
 
 
 class FluorescenceDetector(StandardReadable):
     def __init__(self, prefix: str, name: str = ""):
         with self.add_children_as_readables():
-            self.pos = epics_signal_r(
-                FluorescenceDetectorControlState, prefix + "-EA-FLU-01:CTRL"
-            )
+            self.pos = epics_signal_rw(InOut, prefix + "-EA-FLU-01:CTRL")
```

Comment:
I thought we had in our docs that the prefix should be split to allow some degree of disparate PV prefixes, but I can't find it right now.

Any objection while you're changing this to make the prefix that is passed in `f"{PREFIX.beamline_prefix}-EA-FLU-01:"` for i03 and the signal `prefix + "CTRL"`?